projekt_do_pracy_na_labach/fiszki/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for fiszki.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3  # Ustawienie liczby replik
+
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  tag: "latest"  # Ustawienie tagu obrazu kontenera
+
+backend:
+  port: 8080  # Port, na którym aplikacja backend będzie wystawiona
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: true
+  automount: true
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+podLabels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP  # Typ usługi
+  port: 80
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+
+resources: {}
+  # limits:
+  #   cpu: 200m
+  #   memory: 256Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+livenessProbe:
```

Comment:
Jak podajesz w `deployment.yaml` specyfikację livenessProbe i readinessProbe, to polecałbym to uzupełnić o ścieżkę do endpointu który podaje te informacje